### DISCUSSION TITLE
2D坐标点缺失

### DISCUSSION THREAD

**Discussion by sunmengnan** [Nicholasli1995/EvoSkeleton#53] [Q&A] [answered]
如果2D坐标点是遮挡或不见得，输入网络的时候对应的xy坐标置（0，0）吗?

**Comment by Nicholasli1995** [comment, marked as answer]
> 如果2D坐标点是遮挡或不见得，输入网络的时候对应的xy坐标置（0，0）吗?

预训练的模型没有这个功能。要实现这个功能可以在训练的时候随机把一部分输入点置（0，0）。这样训练出来的模型就可以按你的方法用。
